packages/integration-sdk-runtime/src/storage/memory.test.ts

```diff
@@ -201,9 +201,9 @@ describe('#InMemoryGraphObjectStore', () => {
 
     const e1 = createTestEntity();
     const e2 = createTestEntity();
-
-    await store.addEntities(uuid(), [e1, e2]);
-    store.flushEntities([e1]);
+    const stepId = uuid();
+    await store.addEntities(stepId, [e1, e2]);
+    store.flushEntities([e1], stepId);
```

Comment:
Hm, a little awkward here - it looks like the value of `store.getTotalEntitySizeInBytes()` would be zero (even though there is actually one entity in the store, and thus the size in bytes is >0).

You made a previous suggestion while we talked in-person about actually decrementing the size in the `flushEntities()` function - and I recommended this optimization.

However, I now wonder if your initial proposal, decrementing the size inside of `flushEntities()` itself, is worth implementing. 

Let's discuss. At the very least, we should add a `TODO` note in `getTotalEntitySizeInBytes()` and `getTotalRelationshipSizeInBytes()` about the out-of-sync issue between `entityCount` and `entitySizeInBytes`, or maybe we just should decrement inside of `flushEntities()`.

Reply:
Could be an improvement in terms of consistency but its slower. I don't see that we are using `flushEntities` or `flushRelationships` anywhere else.
If the method flushes all entities for a step, always, then i def makes sense to have it how it is now.
We could also create a new method called `flushEntitiesOfStep` that includes the stepId and it removes all the entities using the stepId.  And we modify the original to measure the size and decrement.